### PR TITLE
ref(sampling): Update copies - [TET-221]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Update sampling onboarding copies according to the latest changes

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

**Review by matejminar — APPROVED** [review]

File: static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.tsx [hunk 1]
```diff
@@ -130,22 +129,16 @@ export function RecommendedStepsModal({
...
       <Header closeButton>
-        <h4>{t('Recommended next steps\u2026')}</h4>
+        <h4>{t('Next steps')}</h4>
       </Header>
       <Body>
         <List symbol="colored-numeric">
           {!!recommendedSdkUpgrades.length && (
             <ListItem>
-              <h5>{t('Update the following SDK versions')}</h5>
+              <h5>{t('Update your SDK(s) before defining sample rules')}</h5>
```
Comment (Jesse-Box) [inline static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.tsx:138]: The copy they suggested from a UX perspective is wrong, so lets back the original copy `Update the following SDK versions`

File: static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.tsx [hunk 2]
```diff
@@ -172,10 +165,10 @@ export function RecommendedStepsModal({
             </ListItem>
           )}
           <ListItem>
-            <h5>{t('Increase your SDK Transaction sample rate')}</h5>
+            <h5>{t('Update your SDK(s) transaction sample rate')}</h5>
```
Comment (Jesse-Box) [inline static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.tsx]: This should be `Increase your client-side transaction sample rate`

File: tests/js/spec/views/settings/project/server-side-sampling/modals/recommendedStepsModal.spec.tsx [hunk 3]
```diff
@@ -120,7 +122,9 @@ describe('Server-side Sampling - Recommended Steps Modal', function () {
     ));
 
     expect(
-      screen.queryByRole('heading', {name: 'Update the following SDK versions'})
+      screen.queryByRole('heading', {
+        name: 'Update your SDK(s) before defining sample rules',
```
Comment (Jesse-Box) [inline tests/js/spec/views/settings/project/server-side-sampling/modals/recommendedStepsModal.spec.tsx:124]: Guess this needs to change based on my second comment

**Review by Jesse-Box — CHANGES_REQUESTED** [review]
Just a few things ;)

**Review by Jesse-Box — APPROVED** [review]

**Pull request merged**